src/uci.cpp

```diff
@@ -151,6 +160,9 @@ void UCI::loop(int argc, char* argv[]) {
   Position pos;
   string token, cmd;
 
+  // implied ucinewgame before the first position command.
```

Comment:
I'd prefer trailing comment and withouth final dot:

`newgame();  // Implied ucinewgame before the first position command`